scripts/modules/elastic_stack.py

```diff
@@ -1175,6 +1068,8 @@ def __init__(self, **options):
                 self.environment["XPACK_FLEET_REGISTRYURL"] = url
             if use_local_package_registry:
                 self.depends_on["package-registry"] = {"condition": "service_healthy"}
+            if self.at_least_version("8.0"):
+                self.environment["XPACK_FLEET_PACKAGES"] = '[{"name":"apm","version":"latest"}]'
```

Comment:
Do we want to use the package version that matches the stack version here?

Reply:
Ideally, yes. At the moment it's 8.0.0-dev1. That tag might change over time (apparently that's what we should have been doing when updating storage), so I don't think we can predict the correct version to use.

I think we can defer this until 8.0 is frozen, and then set the version to "8.0.0" for the 8.0.x releases. When the package is bundled with Kibana we should have a method of telling Kibana to install the bundled version.

Reply:
I believe it is ok to test with the latest available patch version, and from what I understand the apmpackages >= 8.1 should not match if we set the minimum version in the package accordingly. So it would always fetch the latest available 8.0 package then.